optimization/+DoseObjectives/matRad_DoseObjective.m

```diff
@@ -60,6 +60,10 @@
             s = struct@matRad_DoseOptimizationFunction(obj);
             s.penalty = obj.penalty;
         end
+        
+        function constr = convertToConstraint(obj,maxObj,epsilon)
```

Comment:
See my comment about "obj" and "constr" before. Really name it "objective" here.

Reply:
And why is this function in the objective? Couldn't I just pass the Objective to, for example, the constructor of the ObjectiveConstraint directly? Why do I need it?